### ISSUE TITLE
[BUG] `wash build` always generates keys in the local folder instead of `$HOME/.wash/keys

### ISSUE THREAD

**Issue by brooksmtownsend** [wasmCloud/wasmCloud#1624]
### Affected project(s)

- [ ] documentation
- [ ] examples
- [ ] wasmCloud host
- [X] wasmCloud CLI (wash)
- [ ] wasmCloud dashboard UI (washboard)
- [ ] capability providers
- [ ] provider bindgen
- [ ] control interface client
- [ ] other / not sure

### Describe the bug

I've noticed recently `wash` stopped generating and sourcing keys from `$HOME/.wash/keys`, and instead opts to generate keys local to the component folder. I could see this being accepted behavior for the module key itself, but for my account key I don't want to regenerate that for each project.

### Steps to reproduce

```bash
wash new actor --template-name hello-world-rust hello
cd hello
wash build
ll ./keys
```

### Expected behavior

I expect for new keys to be generated and stored at `$HOME/.wash/keys` instead of relative in the local actor folder unless I specify to do that explicitly in `wasmcloud.toml`. 

### Environment

- OS: MacOS
- Shell: zsh
- wasmcloud Version: 0.82.0
- wash Version: 0.26.0

**Comment by brooksmtownsend** [comment]
For folks looking to take this on, see the TODOs added in #1628 for where we should be defaulting to `~/.wash/keys` 😄

**Comment by siddharthkhonde** [comment]
hi @brooksmtownsend , is this issue still open? I am planning to take this up as my first issue

**Comment by brooksmtownsend** [comment]
Closed by #1837
